core/storage/email/gae_models.py

```diff
@@ -365,8 +366,9 @@ def get_by_reply_to_id(cls, reply_to_id):
             reply_to_id: str. The unique 'reply-to' id.
 
         Returns:
-            str or None. The FeedbackEmailReplyToIdModel instance corresponding
-                to the given 'reply-to' id if it is fetched else None.
+            FeedbackEmailReplyToIdModel or None. The instance corresponding to
+            the given 'reply-to' id if it is present in the datastore.
+            Otherwise, None.
```

Comment:
Maybe, it will be better to state explicitly what "Otherwise" is. 

I think it will be better to say "if it is present in the datastore, else None."

Reply:
All fixed !